src/pathfora.js

```diff
@@ -2236,20 +2247,27 @@
         accountId,
         '/user/_uids/',
         seerId,
-        widget.recommend.ql.raw ? '?ql=' + widget.recommend.ql.raw : ''
+        filter ? '?ql=' + filter : ''
       ].join('');
 
-
       this.getData(recommendUrl, function (json) {
         var resp = JSON.parse(json);
         if (resp.data && resp.data.length > 0) {
-          callback(resp.data[0], widget);
+          callback(resp.data);
         } else {
-          callback(null, widget);
+          callback([]);
         }
       }, function () {
-        callback(null, widget);
+        callback([]);
       });
+    },
+
+    /**
+     * @description Construct filter for inline content recommendations
```

Comment:
Added some comments here and a few examples of valid url patterns. Unfortunately I don't think we have any real public docs on this yet. :/